backend/adapter_processor/models.py

```diff
@@ -80,10 +85,24 @@ class AdapterInstance(BaseModel):
         default=False,
         db_comment="Is the adapter instance currently being used",
     )
+    shared_to_org = models.BooleanField(
+        default=False,
+        db_comment="Is the adapter shared to entire org",
+    )
+
+    is_friction_less = models.BooleanField(
+        default=False,
+        db_comment="Does the adapter created through frictionless onboarding",
```

Comment:
```suggestion
        db_comment="Was the adapter created through frictionless onboarding",
```
This suggestion might require a migration though, please check